### ISSUE TITLE
Serial Terminal Wrapping

### ISSUE THREAD

**Issue by CarsonGracey** [parallaxinc/BlocklyProp#419]
![serial terminal](https://cloud.githubusercontent.com/assets/20099916/17065414/83458c54-4ff6-11e6-8761-6491c4a91a51.PNG)

Currently the message output to the serial terminal doesn't wrap at all without a carriage return. It would limit it to print within the actual terminal size.

**Comment by PropGit** [comment]
@CarsonGracey - This is probably because in Issue #321 (specifically in [this post](https://github.com/parallaxinc/BlocklyProp/issues/321)) where I specified that it should make each line capable of displaying up to 256 charaters (corresponding to columns 0 through 255) with a horizontal scroll bar displayed when there's more to see on the right side.  This is with the intention that the terminal window be resizable, font be resizable, and organized displays (ie: field_name: value) can use this space with the cursor control characters (blocks) to make display-panel views for debugging purposes.

In other software, we also added the design of an activity indicator (arrows that appear pointing in the direction that new off-screen activity is occurring) so the user knows that more information is in that direction.

Right now, the terminal is in a state of flux.

If we had it always wrap to the pane, the display-panel views of incoming information would be more limited (and breakable) if the user resized font, resized pane, etc.
